bin/test-merges.js

```diff
@@ -155,8 +155,8 @@ function exportToLocalMC(commitId) {
     // use echo.
     shelljs.exec(`
       echo yes | \
-      env AS_PMD_GIT_DIR=${AS_PMD_GIT_DIR} SYMLINK_TESTS=false ENABLE_MC_AS=1 \
-      bash ${PREPARE_MOCHITESTS_DEV}`,
+      env AS_GIT_BIN_REPO=${AS_GIT_BIN_REPO} SYMLINK_TESTS=false ENABLE_MC_AS=1 \
+      ${PREPARE_MOCHITESTS_DEV}`,
```

Comment:
nit: could you indent these to better indicate they're part of one big command even though the `\` and `exec` imply that?